### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by Helen-Mobify**
* Add addSearcher method documentation and usage example
* Fixed replace method usage example

Reviewer: @donnielrt @fractaltheory

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -164,6 +164,39 @@ descript
   });
 ```
 
+### Adding your own searcher
+
+You can add your own searcher
```
Comment (donnielrt) [inline README.md]: @Helen-Mobify can you add some details about what this does please? It's not clear what a "searcher" is in this context.

File: README.md [hunk 2]
```diff
@@ -164,6 +164,39 @@ descript
...
+| Parameter name | Description |
+|----------------|-------------|
+| **searcher name** | The name of the searcher |
+| **filter function** | A function that should return **true** or **false**. |
+
+#### `Filter function`
+
+| Parameter name | Description |
+|----------------|-------------|
+| **$script** (argument 1) | The current script to be evalutaed |
```
Comment (donnielrt) [inline README.md]: small typo here

**Comment by Helen-Mobify** [comment]
@donnielrt Updated

**Comment by donnielrt** [comment]
@fractaltheory @haroldtreen we probably don't need a release for this, Imma just merge into master unless there're any objections?

**Comment by fractaltheory** [comment]
Seems fine to me.

**Pull request merged**